scripts/factory-state-lib.mjs

```
@@ -3193,7 +3193,12 @@ export function buildFactorySnapshot(paths, { nowMs = Date.now() } = {}) {
         if (!new Set(["needs-ticket-ok", "queued", "parked", "awaiting-morning-review"]).has(job.stage)) {
           throw new Error(`Factory job ${job.id} cannot transfer chat custody while it is ${job.stage}.`);
         }
-        if (String(event.payload.priorStage || "") !== job.stage
+        const transferPriorStage = job.stage === "queued"
+          && job.approvalExpiresAt
+          && Date.parse(job.approvalExpiresAt) <= Date.parse(event.timestamp)
+          ? "needs-ticket-ok"
+          : job.stage;
```

Biome (2.5.6)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)